src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -268,6 +268,10 @@
     "DESCRIPTION": "Draw !M! card. NL This turn, your next attack is played !stsjorbsmod:MetaMagic! additional time.",
     "UPGRADE_DESCRIPTION": "Draw !M! card. NL This turn, your next attack is played !stsjorbsmod:MetaMagic! additional times."
   },
+  "stsjorbsmod:Godsbane": {
+    "NAME": "Godsbane",
+    "DESCRIPTION": "Deal !M! % of targets max HP."
```

Comment:
I would add an apostrophe "target's" and the word "as damage" after the HP.

If you use !D!, will it show the actual damage value for a given target? That seems useful so people know the exact number without having to guess or math our implementation. Altogether that might lead to a DESCRIPTION like

Deal !M!% (!D!) of target's max HP as damage.

Reply:
adding the !D! just shows 0 on the card. Though I imagine there is a way to have it update when you mouse over a target, similar to the update if the target is vulnerable. Not sure of how to make it do that though.

Reply:
It was a good go. Let's call that a future feature.

Reply:
Hey Panic-Point, you can use some features of the CustomJorbsmodCard class to calculate the damage for these kinds of purposes. Look at a card like Drain Life for how to implement it. For a card like this, you can set the baseDamage to 0, and then in method calculateBonusBaseDamage you calculate the percentage of the target's HP. This allows you to use the object's damage property within your use method, and allows you to refer to !D! in an extended description. 
This is a great feature to get used to, especially for displaying calculated values on a card.

Reply:
Thanks for helping find this Cartopol. AbstractDungeon.getCurrRoom().monsters.hoveredMonster seemed to do the trick for showing the actual damage that will be dealt.